CanvasPlusPlayground.xcodeproj/project.pbxproj

```diff
@@ -25,6 +29,11 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
+		9B350F032C95F013003FC60D /* StringExtensions.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = StringExtensions.swift; sourceTree = "<group>"; };
```

Comment:
Is this needed?

Reply:
Apologies, I thought I removed all the references to this file. Will fix

Reply:
Fixed here
db21b3b53fd77fb9739a02bbd676e296987d88e0